tests/test_service.py

```diff
@@ -0,0 +1,126 @@
+"""Tests for service.py ."""
+
+try:
+    from unittest.mock import create_autospec, Mock, patch, MagicMock
+except ImportError:
+    from mock import create_autospec, Mock, patch, MagicMock
+
+from reportportal_client.service import _convert_string, _list_to_payload, \
+    uri_join, _get_id, _get_msg, _get_data, _get_json, _get_messages, \
+    ReportPortalService
+
+import unittest
+from datetime import datetime
+
+
+class TestServiceFunctions(unittest.TestCase):
+    """Test for additional functions."""
+
+    def test_check_convert_to_string(self):
+        """Test for support and convert strings to utf-8."""
+        self.assertEqual(_convert_string("Hello world"), 'Hello world')
+        self.assertEqual(type(_convert_string("Hello world")), str)
+
+    def test_list_to_payload(self):
+        """Test convert dict to list of dicts."""
+        initial_dict = {'key': "value", 'key1': 'value1'}
+        expected_list = [{'key': 'key', 'value': 'value'},
+                         {'key': 'key1', 'value': 'value1'}]
+        self.assertEqual(_list_to_payload(initial_dict), expected_list)
+
+    def test_get_id(self):
+        """Test for get id from Response obj."""
+        fake_json = {"id": 123}
+
+        with patch('requests.Response', new_callable=MagicMock()) as mock_get:
+            mock_get.status_code = 200
+            mock_get.json.return_value = fake_json
+
+            obj = _get_id(mock_get)
+
+        self.assertEqual(obj, 123)
+
+    def test_get_msg(self):
+        """
+        Test get_msg recieved from Response.
+
+        :return dict
```

Comment:
Your test function does not return anything(Well, actually it returns None). 

"""Test get_msg recieved from Response."""